torch/nn/modules/module.py

```diff
@@ -464,6 +470,43 @@ def convert(t):
 
         return self._apply(convert)
 
+    @classmethod
+    def register_global_backward_hook(cls, hook):
+        r"""Registers a backward hook common to all the modules.
```

Comment:
If this stays a global flag, I would add a Note or Warning here explaining that this sets a global state and the user should avoid using these if possible.
Recommended use cases is for debugging issues with nn.Module to have a better idea when they are called.

Same comment for the other two.